### PR TITLE
Construct memoized debounce

### PR DESCRIPTION
**Pull request by baileytincher**
updateOutput and editCell had a bug resulting in different cells replacing the debounced function call

### CODE REVIEW

File: client/src/socket/client.ts [hunk 1]
```diff
@@ -211,9 +211,18 @@ export class ActuallyColabSocketClient extends EventEmitter<ActuallyColabEventLi
...
    * @param cell_id Cell to edit.
+   * @param cellData Cell data to replace with
    */
-  public unlockCell = (nb_id: Notebook['nb_id'], cell_id: DCell['cell_id']): void => {
-    this.editCell.flush();
+  public unlockCell = (
+    nb_id: Notebook['nb_id'],
+    cell_id: DCell['cell_id'],
+    cellData: {
+      contents: DCell['cell_id'];
```
Comment (jtaylorchang) [inline client/src/socket/client.ts:220]: I think this is supposed to be `DCell['contents']`

File: client/src/socket/client.ts [hunk 2]
```diff
@@ -226,20 +235,22 @@ export class ActuallyColabSocketClient extends EventEmitter<ActuallyColabEventLi
...
-  public editCell = debounce(
+  public editCell = memoizeDebounce(
     (
       nb_id: Notebook['nb_id'],
       cell_id: DCell['cell_id'],
       cellData: {
-        contents?: DCell['cell_id'];
-        language?: DCell['language'];
-        cursor_pos?: DCell['cursor_pos'];
+        contents: DCell['cell_id'];
```
Comment (jtaylorchang) [inline client/src/socket/client.ts:243]: Same here

File: server/src/socket/cell/edit-cell.ts [hunk 3]
```diff
@@ -19,7 +19,7 @@ interface TEditCellEventBody {
     cellData: {
       contents?: DCell['cell_id'];
```
Comment (jtaylorchang) [inline server/src/socket/cell/edit-cell.ts:20]: Same here

File: client/src/socket/client.ts (hunk 2 above)
Comment (baileytincher) [reply client/src/socket/client.ts:243]: lol never noticed

**Pull request merged**